paraview/xmls/SignedDistanceField.xml

```diff
@@ -0,0 +1,99 @@
+<ServerManagerConfiguration>
+  <ProxyGroup name="filters">
+    <SourceProxy
+      name="ttkSignedDistanceField"
+      class="ttkSignedDistanceField"
+      label="TTK SignedDistanceField">
+      <Documentation
+        long_help="TTK plugin for the signed distance field computation of surfaces."
```

Comment:
Also, please describe the difference backends here (with references). Thanks!

Reply:
Done in the next commit!